styles/style.css

```diff
@@ -64,20 +65,235 @@ button {
 }
 
 .left__menu__item__dropdown__dropdown {
+  display: flex;
   flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 5px;
   margin-left: 200px;
   margin-top: -42px;
   border: 1px solid beige;
   border-radius: 5px;
-  gap: 20px;
   padding: 10px;
   display: none;
   position: absolute;
   z-index: 1;
   background: linear-gradient(345deg, rgb(28, 34, 40) 0%, rgba(238, 223, 198, 0.7665660014) 45%, rgb(238, 208, 138) 100%);
 }
 
+.right__menu {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 0;
+  width: 100vw;
+}
+
+footer {
+  display: flex;
+  position: fixed;
+  left: 0;
+  height: 50px;
+  width: 100%;
+  background: linear-gradient(39deg, rgb(80, 52, 35) 5%, rgb(227, 142, 0) 69%, rgb(235, 98, 0) 100%);
+  color: white;
+  border-radius: 10px;
+  margin: 0px 10px 5px 10px;
+  bottom: 0;
+}
+
+.apps__img {
+  height: 40px;
+  width: 40px;
+  border-radius: 5px;
+}
+
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  left: 0;
+  top: 0;
+  height: 100%; /* Full height */
+  width: 100%; /* Full width */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0); /* Fallback color */
+  background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
+}
+
+/* Modal Content/Box */
+.modal-content {
+  background-color: #fefefe;
+  margin: 15% auto; /* 15% from the top and centered */
+  width: 80%; /* Could be more or less, depending on screen size */
+  height: 60%;
+  color: black;
+  border: 2px solid grey;
+  border-radius: 10px;
+}
+.modal-content--calculator {
+  height: 49%;
+  width: 20%;
+  background-color: rgb(91, 84, 80);
+  color: white;
+  padding: 3px;
+  font-size: 25px;
+}
+
+.modal__header {
+  background-color: antiquewhite;
+  height: 20px;
+  text-align: center;
+  padding: 5px;
+  margin: 0;
+}
+.modal__header--calculator {
+  background-color: rgb(91, 84, 80);
+}
+.modal__header--chrome {
+  height: 25px;
+  text-align: left;
+  background-color: rgb(231, 234, 237);
+}
+.modal__header--chrome--icon {
+  font-size: 15px;
+}
+.modal__header--chrome--newTab {
+  width: 180px;
+  border: 2px solid rgb(231, 234, 237);
+  border-radius: 10px;
+  border-bottom: none;
+  background-color: white;
+  display: inline-block;
+  padding: 4px;
+  margin-left: 10px;
+}
+.modal__header--chrome--newTab--close {
+  margin-left: 70px;
+}
+
+.modal__text {
+  width: 100%;
+  height: 100%;
+}
+
+.modal__text--calculator--input {
+  text-align: end;
+  height: 52px;
+  font-size: 50px;
+}
+.modal__text--calculator--operators {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 1px;
```

Comment:
grid might allow you to do some more nuanced things here, like remove the gap and have the 0 button take up two columns etc